lib/facility_factory.rb

```diff
@@ -0,0 +1,89 @@
+class FacilityInfo
+    attr_reader :geom, :id, :office, :address_1, :address_2,
+                :city, :state, :zip, :phone, :hours, :services,
+                :parking, :photo, :address_id, :comp_region, 
+                :comp_region_2, :comp_region_3, :county, 
+                :office_type, :fax, :email, :texting_num, :size,
+                :region_num #:format_hours
+    def initialize(details)
+        @geom = details[:geom]
+        @id = details[:id]
+        @office = details[:office]
+        @address_1 = details[:address_1]
+        @address_2 = details[:address_2]
+        @city = details[:city]
+        @state = details[:state]
+        @zip = details[:zip]
+        @phone = details[:phone]
+        @fax = details[:fax]
+        @email = details[:email]
+        @texting_num = details[:texting_num]
+        @hours = details[:hours]
+        #####
+        #@format_hours = details[:ny_hours]
+        #####
+        @services = details[:services]
+        @parking = details[:parking]
+        @photo = details[:photo]
+        @address_id = details[:address_id]
+        @comp_region = details[:comp_region]
+        @comp_region_2 = details[:comp_region_2]
+        @comp_region_3 = details[:comp_region_3]
+        @office_type = details[:office_type]
+        @county = details[:county]
+        @size = details[:size]
+        @region_num = details[:region_num]
+    end
+
+    # private
+
+    # def ny_hours(details)
+    #     days = %w[monday tuesday wednesday thursday friday]
+    #     hours = days.map do |day|
+    #         start_key = "#{day}_beginning_hours".to_sym
+    #         end_key = "#{day}_ending_hours".to_sym
+    #         start = details[start_key]
+    #         ending = details[end_key]
+    #         "#{day.capitalize}: #{start} - #{ending}" if start && ending
+    #     end.compact
+    #     hours.join(', ')
+  
+    # end
+
+end
+
+class FacilityFactory
```

Comment:
oh, interesting. So every class should have it's own file. We don't want multiple classes in one file. I don't think you need the Facility Info class though, all that ingestion of data should happen in the FacilityFactory